src/overlays/actors/ovl_Dm_Char08/z_dm_char08.c

```diff
@@ -28,71 +63,1061 @@ const ActorInit Dm_Char08_InitVars = {
     (ActorFunc)DmChar08_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80AB1764[] = {
+#include "overlays/ovl_Dm_Char08/ovl_Dm_Char08.c"
+
+static AnimationInfo sAnimationInfo[] = {
+    { &gTurtleIdleAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+    { &gTurtleSwimAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+    { &gTurtleFloatAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+    { &gTurtleSpeak1Anim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+    { &gTurtleCoughAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+    { &gTurtleSpeak2Anim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+    { &gTurtleYawnAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -24.0f },
+};
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 4000, ICHAIN_STOP),
 };
 
-#endif
+void DmChar08_UpdateEyes(DmChar08* this) {
+    switch (this->eyeMode) {
+        case TURTLE_EYEMODE_0:
+            this->eyeIndex = 0;
+            if (this->blinkTimer > 0) {
+                this->blinkTimer--;
+            } else {
+                this->blinkTimer = 0;
+            }
+            if (this->blinkTimer < 4) {
+                this->eyeIndex = this->blinkTimer;
+            }
+            if (this->blinkTimer == 0) {
+                this->blinkTimer = Rand_S16Offset(30, 30);
+            }
+            break;
+
+        case TURTLE_EYEMODE_1:
+            this->eyeIndex = 4;
+            if (this->blinkTimer > 0) {
+                this->blinkTimer--;
+            } else {
+                this->blinkTimer = 0;
+            }
+            if (this->blinkTimer < 4) {
+                this->eyeIndex = this->blinkTimer + 4;
+            }
+            if (this->blinkTimer == 0) {
+                this->blinkTimer = Rand_S16Offset(30, 30);
+            }
+            break;
+
+        case TURTLE_EYEMODE_2:
+            this->eyeIndex = 2;
+            break;
+
+        case TURTLE_EYEMODE_3:
+            this->eyeIndex = 4;
+            break;
+
+        case TURTLE_EYEMODE_5:
+            this->eyeIndex = 8;
+            break;
```

Comment:
Maybe you can't name all of them, but some of these can be named, right? Since they just set a static `eyeIndex`.

Reply:
Wasn't too bad just naming them all.